changelog_unreleased/javascript/13157.md

```diff
@@ -0,0 +1,31 @@
+#### Fix line breaks in multiline strings aren't replaced with literalline and confuse fits (#13157 by @thorn0 )
```

Comment:
The thing in parens should the PR number and author, i.e. you. Also the file should be named after the PR.